src/db.ts

```diff
@@ -367,25 +383,33 @@ export const findPersonByAccountIBAN = (iban) =>
   findPersonByAccountField((person) => person.account.iban === iban);
 
 export const getWebhooks = async () => {
-  const webhooks = await redisClient
-    .keysAsync(`${process.env.MOCKSOLARIS_REDIS_PREFIX}:webhook:*`)
-    .then((keys) => {
-      if (keys.length < 1) {
-        return [];
-      }
-      return redisClient.mgetAsync(keys);
-    })
-    .then((values) => values.map(JSON.parse));
-
-  return webhooks;
+  let cursor = 0;
```

Comment:
:question: are `keys` and `mget`  not supported?

:thinking: the new implementation is much harder to read than the old one. Perhaps if `keys` and `mget` are not supported, can we abstract it?